COMMON_ISSUES.md

```diff
@@ -72,6 +77,11 @@ issues on the Firefox bug tracker:
 * [Bug 881532](https://bugzilla.mozilla.org/show_bug.cgi?id=881532)
 * [Bug 1278384](https://bugzilla.mozilla.org/show_bug.cgi?id=1278384)
 
+Also, there is an existing bug in twilio-video.js where publication of
+DataTracks after joining a group Room never reaches completion. While we work
+to fix this bug, you can work around this by publishing your DataTracks
+while connecting to a Room using the `tracks` property of ConnectOptions.
```

Comment:
Similar question as above. I think our hypothesis is that this is a media server bug. Also, small nitpick: `s/group Room/Group Room/`.